libs/components/filter-bar/src/lib/modules/filter-bar/models/filter-item-modal-instance.ts

```diff
@@ -0,0 +1,49 @@
+import { inject } from '@angular/core';
+
+import { Subject } from 'rxjs';
+
+import { SkyFilterBarFilterValue } from './filter-bar-filter-value';
+import { SkyFilterItemModalContext } from './filter-item-modal-context';
+
+/**
+ * A specialized `SkyModalInstance` wrapper.
+ */
+export class SkyFilterItemModalInstance<TData = Record<string, unknown>> {
+  /**
+   * The context provided to the filter modal component.
+   */
+  public readonly context = inject<SkyFilterItemModalContext<TData>>(
+    SkyFilterItemModalContext,
+  );
+
+  readonly #canceled = new Subject<SkyFilterBarFilterValue | undefined>();
+  readonly #saved = new Subject<SkyFilterBarFilterValue | undefined>();
+
+  /**
+   * Emits when the user cancels the modal.
```

Comment:
```suggestion
   * Fires when the user cancels the modal.
```
Probably doesn't matter too much since this is marked internal, but "emits" seems like the wrong word here. I think we usually use "fires" to indicate that an action (like cancelling the modal) triggers an event and that we reserve "emit" for describing when the event sends out data or something.